obs_img_utils/cli.py

```diff
@@ -177,6 +179,35 @@ def main(context):
     is_flag=True,
     help='Skip the image checksum validation.'
 )
+@click.option(
+    '--add-conditions-file',
+    type=click.STRING,
+    help='Specify conditions for the image from a file (with the conditions '
+         'in json format as a LIST)',
+    default=''
+)
+@click.option(
+    '--add-conditions-arg',
+    type=click.STRING,
+    help='Specify conditions for the image from a CLI arg (with the '
+         'conditions in json format as a LIST as a single string)',
+    default=''
+)
+@click.option(
+    '--disallow-licenses-arg',
+    type=click.STRING,
+    help='Specify any licenses that should not be in the image.'
+         'More than one license can be specified separated by comma(,)',
+    default=''
+)
+@click.option(
+    '--disallow-packages-arg',
+    type=click.STRING,
+    help='Specify any packages that should not be in the image.'
+         'More than one license can be specified separated by comma(,). This '
+         'can use a wildcard(*) to mach any naming pattern like "*-mini".',
+    default=''
+)
```

Comment:
Can the `--disallow-licenses-arg` and `--disallow-packages-arg` not be expressed as a condition? If they can then this would be duplication of information intake. If they cannot maybe the conditions can be expanded to handle this use case?